src/components/app/details/cdDetails/CDDetails.tsx

```diff
@@ -75,6 +76,12 @@ export default function CDDetails() {
             setHasMore(true)
             setHasMoreLoading(true)
         }
+        let queryString = new URLSearchParams(location.search)
+        let queryParam = queryString.get('type')
+        let deploymentStageType = queryParam === "PRECD" ? "PRE" : "POST"
```

Comment:
Fix this condition. this will not work in case of deploy

Reply:
done